device.mk

```diff
@@ -391,7 +390,18 @@ PRODUCT_COPY_FILES += \
     device/lge/hammerhead/ubuntu/ubuntu-location-service.conf:system/ubuntu/etc/init/ubuntu-location-service.conf \
     device/lge/hammerhead/ubuntu/display.conf:system/ubuntu/etc/ubuntu-touch-session.d/android.conf \
     device/lge/hammerhead/ubuntu/display.conf:system/ubuntu/etc/ubuntu-touch-session.d/hammerhead.conf \
-    device/lge/hammerhead/ubuntu/ofono.override:system/ubuntu/etc/init/ofono.override
+    device/lge/hammerhead/ubuntu/ofono.override:system/ubuntu/etc/init/ofono.override \
+    device/lge/hammerhead/ubuntu_overlay/system/etc/init/bluetooth-touch-hammerhead.conf:system/ubuntu-overlay/system/etc/init/bluetooth-touch-hammerhead.conf \
+    device/lge/hammerhead/ubuntu_overlay/system/etc/init/ofono.override:system/ubuntu-overlay/system/etc/init/ofono.override \
+    device/lge/hammerhead/ubuntu_overlay/system/etc/wifi/bcmdhd.cal:system/ubuntu-overlay/system/etc/wifi/bcmdhd.cal \
+    #device/lge/hammerhead/ubuntu_overlay/system/etc/acdbdata:system/ubuntu-overlay/system/etc/acdbdata \
+    #device/lge/hammerhead/ubuntu_overlay/system/etc/mixer_paths.xml:system/ubuntu-overlay/system/etc/mixer_paths.xml \
+    device/lge/hammerhead/ubuntu_overlay/system/usr/bin/hciattach:system/ubuntu-overlay/system/usr/bin/hciattach \
+    device/lge/hammerhead/ubuntu_overlay/system/usr/share/bluetooth-touch/hammerhead:system/ubuntu-overlay/system/usr/share/bluetooth-touch/hammerhead
+
```

Comment:
Your PR is named "charging animation" but here you also change a lot of the overlay stuff. This should not be part of the PR I think. Also, I think those files are already copied elsewhere, otherwise certain things would not work?